### PR TITLE
add info about 0x801c004D

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/6505

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by mypil** [comment]
@mapalko @Dansimp - PR has been copy edited and is ready for final review. This is based on issue #6505.

Thank you.

cc: @kenwith

**Comment by mapalko** [comment]
According to the AAD Dev team this error is DSREG_NO_DEFAULT_ACCOUNT. This  means that NGC provisioning we were unable to find the default WAM account to use to request AAD token for provisioning. There was a bug on this that was resolved in Vibranium. Do you think any of that information is helpful or does is not matter since, it requires a bug fix anyway?

**Comment by illfated** [comment]
Looks like very useful information, because it explains why the device failed to register, which I think most admins would want to know. Then again, that is my own perspective.

**Comment by mypil** [comment]
@mapalko - I agree with @illfated that this information is a useful addition to the doc. Please let us know if there's a need to modify what is on this PR so we can let @VLG17 make the necessary changes if required. Thank you.

File: windows/security/identity-protection/hello-for-business/hello-errors-during-pin-creation.md [hunk 1]
```diff
@@ -98,6 +98,7 @@ For errors listed in this table, contact Microsoft Support for assistance.
 | 0x801C03F0  | ​There is no key registered for the user. |
 | 0x801C03F1  | ​There is no UPN in the token. |
 | ​0x801C044C  | There is no core window for the current thread. |
+| 0x801c004D  | Unable to enroll a device to use a PIN for login. |
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-errors-during-pin-creation.md:101]: Maybe something like this? Feel free to suggest a different phrasing or wording to provide the useful information in a format suited for this table.
```suggestion
| 0x801c004D  | DSREG_NO_DEFAULT_ACCOUNT: NGC provisioning is unable to find the default WAM account to use to request AAD token for provisioning. Unable to enroll a device to use a PIN for login. This bug was resolved in Vibranium. |
```
Maybe this information needs to be split into one part for the table and a separate part to add after the table. I am uncertain what would be most helpful here.

Comment (mypil) [reply windows/security/identity-protection/hello-for-business/hello-errors-during-pin-creation.md:101]: @VLG17  - please merge the commit on the copy/edit review done by @illfated or make the necessary changes on the PR following the suggestions made. Thank you.

Comment (VLG17) [reply windows/security/identity-protection/hello-for-business/hello-errors-during-pin-creation.md:101]: @illfated @mypil I will add it, but i will exclude this bit "This bug was resolved in Vibranium". I don't think the general audience will know about Vibranium (without searching).

Comment (illfated) [reply windows/security/identity-protection/hello-for-business/hello-errors-during-pin-creation.md]: Fine by me. Good decision.

**Comment by mypil** [comment]
@mapalko - For your final review and approval for merging.

Thank you.

**Pull request merged**